webpack.config.js

```diff
@@ -10,5 +10,17 @@ module.exports = {
     globalObject: "this"
   },
   mode: "production",
-  devtool: "source-map"
+  devtool: "source-map",
+    optimization: {
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          output: {
+            keep_classnames: true
+          }
+        }
+      )
```

Comment:
Oops, seems I confused you.. this was the problem:

```suggestion
      })
```